MarsQA-1/SpecflowPages/Pages/LanguagesPage.cs

```diff
@@ -0,0 +1,304 @@
+﻿using MarsQA_1.Helpers;
+using NUnit.Framework;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Support.UI;
+using System;
+using System.Collections.ObjectModel;
+using System.Linq;
+using System.Threading;
+
+namespace MarsQA_1.Pages
+{
+    public static class LanguagesPage
```

Comment:
this should not be a static class please change this, after doing this you will then need to create objects

Reply:
please fix this for every where you have done this!

Reply:
Removed static keyword every where except util and helper classes. And created objects.